### PR TITLE
 Fixes #2 Wrapper Skycoin Api PHP

### LINKED ISSUE #2: PHP API Wrapper for the Skycoin node API
## Expected Behavior

Access to the Skycoin node REST API implemented in Java code or as .jar assembly

## Actual Behavior

Right now this is only possible using code generated by SWIG , which requires memory handles

## Specifications

  - Version: 0.25.1
  - Platform: all
  - Subsystem: all exported API sets

## Possible Solution

Generate with Swagger (share more info, please) . Beware of CSRF .

 - [ ] Use Skycoin node Swagger spec to generate initial Java REST API client
 - [ ] Add make target to automate generation in ./build
 - [ ] Aforementioned target should copy / merge relevant files into source code tree
 - [ ] README explaining the process
 - [ ] Mention in CHANGELOG
 - [ ] Patch the result Swagger-gen REST client to support specifying node address
 - [ ] Update package metadata
 - [ ] Run basic Swagger tests in Travis
 - [ ] Unify testing code if multiple frameworks are used across source tree
 - [ ] Beware of CSRF auth

### PR DESCRIPTION
**Pull request by stdevMac**
Fixes #2 

Changes:
 - Added `lib/skyapi`, a wrapper for Skycoin api

Does this change need to mentioned in CHANGELOG.md?
 - Yes

Need Test

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.PHONY: build
+
+
+build:
+	(cd lib/skyapi && composer install)
+
+
+test:
+	(cd lib/skyapi && vendor/bin/phpunit)
```
Comment (olemis) [inline Makefile]: Add `help` command please

File: lib/skyapi/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM composer:latest
```
Comment (olemis) [inline lib/skyapi/Dockerfile]: Two `Dockerfile`'s , why ?

Comment (stdevMac) [reply lib/skyapi/Dockerfile]: My mistake, i delete it in the last commit

File: Makefile (hunk 1 above)
Comment (stdevMac) [reply Makefile]: done

**Pull request merged**